Volume/Views/BookmarksList.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  BookmarksList.swift
+//  Volume
+//
+//  Created by Cameron Russell on 11/15/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct BookmarksList: View {
+    var body: some View {
+        GeometryReader { geometry in
+            NavigationView {
+                ScrollView(showsIndicators: false) {
+                    Header(text: "Saved Articles")
+                    if let savedArticles = articleData.filter{ $0.saved }, savedArticles.count > 0 {
```

Comment:
Spacer after `filter`. And you probably don't need `savedArticles.count > 0` because your ForEach will just iterate over `[]`. It's not a big deal though, and maybe keeping it is better. I'm not sure.

Reply:
We want an empty view if there are no articles saved, so that's why I added that condition.